package.json

```diff
@@ -8,6 +8,7 @@
     "type": "git",
     "url": "http://github.com/azavea/tilejson-validator.git"
   },
+  "license": "Apache-2.0",
```

Comment:
As discussed, we should also include an actual copy of the license as `LICENSCE.md` in the repo root.